rds/aurora/mysql/cluster/README.md

```diff
@@ -0,0 +1,78 @@
+# Aurora MySQL Cluster
+
+This Acorn creates an Aurora MySQL cluster running on AWS RDS service. This Acorn is best used when you have a steady state workload that can be serviced by a VM instance. If you have a burstable workload you could consider using the Aurora Serverless v2 Acorn.
+
+## Usage
+
+From the CLI you can run the following command to create an Aurora MySQL cluster:
+
+```shell
+acorn run -n rds-mysql-cluster ghcr.io/acorn-io/aws/rds/aurora/mysql/cluster:v1.#.#
+```
+
+From an Acornfile you can use the following Acorn:
+
+```cue
+services: "rds-mysql-cluster": {
+    image: ghcr.io/acorn-io/aws/rds/aurora/mysql/cluster:v1.#.#
+}
+
+containers: wp: {
+    image: wordpress:latest
+    ports: publish: "80/http"
+    env: {
+        WORDPRESS_DB_HOST: "@{services.rds-mysql-cluster.address}"
+        WORDPRESS_DB_USER: "@{services.rds-mysql-cluster.secrets.amdin.username}"
+        WORDPRESS_DB_PASSWORD: "@{services.rds-mysql-cluster.secrets.admin.password}"
+        WORDPRESS_DB_NAME: "instance"
+    }
+}
+```
+
+## Arguments
+
+| Name | Description | Type | Schema |
+|------|-------------|------|--------|
+| adminUsername | Name of the root/admin user. Default is admin. | string |
```

Comment:
You've listed a `Schema` header but each row seems to only fill out the first three columns.

Reply:
Oh and we should probably split the default value out into its own column.